package.json

```diff
@@ -31,6 +32,7 @@
   "license": "MIT",
   "dependencies": {
     "d3": "^5.7.0",
+    "file-system": "^2.2.2",
```

Comment:
Does `package-lock.json` need to be checked-in?

Reply:
package-lock.json does added